internal/vm/hcs/windows.go

```diff
@@ -1,10 +1,12 @@
 package hcs
 
 import (
+	"context"
+
 	hcsschema "github.com/Microsoft/hcsshim/internal/hcs/schema2"
 )
 
-func (uvmb *utilityVMBuilder) SetCPUGroup(id string) error {
+func (uvmb *utilityVMBuilder) SetCPUGroup(ctx context.Context, id string) error {
```

Comment:
why add the context?

Reply:
Need it for the remotevm impl

Reply:
As we need to grab the underlying u64 hypervisor ID by querying HCS